Apps/APN-UIKit/APN UIKit/APN UIKit/View/SettingsViewController.swift

```diff
@@ -70,10 +87,37 @@ class SettingsViewController: UIViewController {
             handler(settings.authorizationStatus)
         }
     }
+    
+    func compareAndSave() {
+        
+        // Track Url
+        if currentSettings.trackUrl != trackUrlTextField.text {
+            storage.trackUrl = trackUrlTextField.text
+        }
+        
+        // Background Queue Seconds Delay
+        if currentSettings.bgQDelay != bgQTakDelayTextField.text {
+            storage.bgQDelay = bgQTakDelayTextField.text
+        }
+        
+        // Min number of tasks
+        if currentSettings.bgQMinTasks != bgQMinTasksTextField.text {
+            storage.bgNumOfTasks = bgQMinTasksTextField.text
+        }
+        
+        // Min tasks in queue
+        if currentSettings.bgQMinTasks != bgQMinTasksTextField.text {
+            storage.bgNumOfTasks = bgQMinTasksTextField.text
+        }
+        
+        // Track screen enabled
+//        if currentSettings.isTrackScreenEnabled != trackScreenToggle.va
+    }
 
     // MARK: - Actions
 
     @IBAction func saveSettings(_ sender: UIButton) {
+        compareAndSave()
         showAlert(withMessage: "Settings saved. This will require an app restart to bring the changes in effect.", action: popToSource)
```

Comment:
Why does the app need to be restarted? Is there a way we can prevent the need for that?

Reply:
Restarting the app is an easy, quick and reliable approach IMO. We had tried to avoid app restart in RH and it led to a bit of complication.

Reply:
This message is a part of new design. Check screenshot below:

Instead of a label, I have added an alert.

<img width="434" alt="Screenshot 2023-05-03 at 4 05 09 PM" src="https://user-images.githubusercontent.com/91549653/235893447-025410f7-2c1e-43e5-b1cc-0281365a21b5.png">

Reply:
I do understand that the design shows text "Editing settings will require an app restart". Label or Alert, I am suggesting we do not have a label or an Alert with this message in it. 

I don't think that the app or the SDK requires an app restart for settings to go into effect. Therefore, I don't see a reason to include a warning like this when I don't think it's true. 

I suggest we leave our this Label/Alert until we notice that the app does need to be restarted for SDK settings to go into effect.

Reply:
Got your point but I have not yet implemented the functionality to make changes in real-time while the app is running. In Remote Habits, we encountered some issues that were only resolved after the app was restarted. Remote Habits had a logic to update configurations while the app was running.

In my opinion, removing the label would only complicate things. I will stick to the original design and add real-time configuration updates if the QA and product teams feel that it's necessary.